### PR TITLE
rank down release notes page in search results

### PR DESCRIPTION
**Pull request by boegel**
cfr. https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#rank-down

This pushes down the release notes page when looking for `PYPI_SOURCE` for example, effectively listing more relevant search results higher up.

### CODE REVIEW

File: .mdlrc [hunk 1]
```diff
@@ -1 +1,3 @@
 style "#{File.dirname(__FILE__)}/.mdl.rb"
+
+ignore_front_matter true
```
Comment (branfosj) [inline .mdlrc:3]:
```suggestion
ignore_front_matter true
style "#{File.dirname(__FILE__)}/.mdl.rb"
```

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+search:
+  boost: 0.5
+---
+
```
Comment (branfosj) [inline docs/release-notes.md:5]:
```suggestion
```

File: docs/release-notes.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+search:
+  boost: 0.5
+---
+
 # EasyBuild release notes {: #release_notes }
```
Comment (branfosj) [inline docs/release-notes.md:6]:
```suggestion
---
# EasyBuild release notes {: #release_notes }

```

**Comment by branfosj** [comment]
@boegel Please pull in #165, as that should fix the linter false positive and allow us to merge this.

**Comment by boegel** [comment]
We should also do this for the API part of the docs

**Comment by branfosj** [comment]
fixes https://github.com/easybuilders/easybuild-docs/issues/134

**Comment by branfosj** [comment]
> We should also do this for the API part of the docs

We'll do this in a follow up PR, as the auto-generated page are more complicated.

**Review by branfosj — APPROVED** [review]

**Pull request merged**